component/kafka/group/option.go

```diff
@@ -26,6 +28,38 @@ func FailureStrategy(fs kafka.FailStrategy) OptionFunc {
 	}
 }
 
+// CheckTopic will attempt to:
+//  1. connect to the broker
```

Comment:
QQ: Since code might change and usually the comments not and since reading the code should be self-explanatory would it make sense not to have such a detailed comment?